packages/vscode/src/plugins/prisma-language-server/index.ts

```diff
@@ -74,10 +86,18 @@ const startFileWatcher = (rootPath: string) => {
     recursive: true,
     ignoreInitial: true,
     ignore: (targetPath) => {
-      if (targetPath === rootPath) {
-        return false
+      if (targetPath === rootPath) return false
+
+      // we ignore all files that are not a match
+      const isIgnored = watchedPaths.reduce((acc, regex) => {
+        return acc && targetPath.match(regex) === null
+      }, true)
```

Comment:
Does that all filter out `node_modules` in subfolders of e.g. pacakges in monorepos? 
I think that was why the minimatch pattern started with `**`.

Reply:
That's a good point. It was previously also broken:
![image](https://github.com/prisma/language-tools/assets/18401805/54816679-db22-4838-923b-095c7fee0050)
The dot in `.pnpm` needed to be escaped it seems, or some other setting that I am not aware of.

Reply:
![Uploading image.png…]()

Reply:
![image](https://github.com/prisma/language-tools/assets/18401805/a8a39d4c-dff0-44d6-87d2-552707859899)

Reply:
How so you want to go about this? Do we explicitly allow `pnpm` or look through all `node_modules` folders and sub-folders for an eventual `.prisma/client` (that's what the glob intended, I think, but also totally non-optimized).

Reply:
that seems... not great, I'd say probably no. Do we not guarantee that the client will always be in the same place (unless someone manually moves it in their repo but I think that's on them at that point?)

Reply:
Whatever works for most of our users.